models/Schematic_DataModel.php

```diff
@@ -49,12 +49,44 @@ public static function fromYaml($yaml, $overrideYaml)
         $data = Yaml::parse($yaml);
         $overrideData = Yaml::parse($overrideYaml);
         if ($overrideData != null) {
-            $mergedData = array_replace_recursive($data, $overrideData);
-        } else {
-            $mergedData = $data;
+            $data = array_replace_recursive($data, $overrideData);
         }
+        $data = Schematic_DataModel::replaceEnvVariables($data);
```

Comment:
Apparently this doesn't work in PHP 5.3 in anonymous functions. Revert this or drop compatibility with 5.3?